modules/javafx.media/src/main/native/gstreamer/3rd_party/libffi/include/ffitarget.h

```diff
@@ -50,7 +50,9 @@
 #endif
 
 #define FFI_TARGET_SPECIFIC_STACK_SPACE_ALLOCATION
-//#define FFI_TARGET_HAS_COMPLEX_TYPE
```

Comment:
In the original release of libffi 3.2.1,  this line is not commented. which means it was commented in JavaFX. I cannot trace when and why was it commented. Should it be reasoned to whether enable it or not ?

Reply:
That's a good question. The commented line was there from the first commit (in 9, changeset 9231:241f9696e3ad, https://bugs.openjdk.java.net/browse/JDK-8043352) but I don't see a reason on why it was disabled.

libffi is also built in the GraalVM project, and that line is not commented there. 

@sashamatveev or @kevinrushforth do you remember the reason?

Reply:
No, I don't remember.

Reply:
No, I don't remember as well.